src/legacy/core_plugins/kibana/migrations/migrations.js

```diff
@@ -285,6 +285,50 @@ function transformFilterStringToQueryObject(doc) {
   }
   return newDoc;
 }
+function transformSplitFiltersStringToQueryObject(doc) {
+  // Migrate split_filters in TSVB objects that were'nt migrated in 7.3
+  // If any filters exist and they are a string, we assume them to be lucene syntax and transform the filter into an object accordingly
+  const newDoc = cloneDeep(doc);
+  const visStateJSON = get(doc, 'attributes.visState');
+  if (visStateJSON) {
```

Comment:
At some point we really should write a utility for that annoying boilerplate code... but def not in this PR :D